### DISCUSSION TITLE
How safe is Windows Update Restored?

### DISCUSSION THREAD

**Discussion by TheKnowGuy** [LegacyUpdate/LegacyUpdate#309] [Questions & Help] [answered]
Hello I have a question I saw that legacy update does not work for Windows 98 and they have a sister project called Windows Update Restored (https://www.windowsupdaterestored.com/) and when I go to the website I get a pop-up from Malwarebytes Web Extension that the Website is not safe and I check that Statistics should I trust this or no?
![Screenshot 2024-11-27 181004](https://github.com/user-attachtments/assets/153f75a3-9d2d-4ea5-8cf5-09aa6c5ebb3b)
![Screenshot 2024-11-27 180943](https://github.com/user-attachments/assets/c55a7537-b1e9-4586-a52a-181307bda65d)
![Screenshot 2024-11-27 180952](https://github.com/user-attachments/assets/6c9e2b60-0859-4e9b-a687-82707917ec57)

**Comment by kirb** [comment, marked as answer]
Almost definitely this should be a false positive. Both Legacy Update and Windows Update Restored run into this sometimes, mostly because they’re doing things that are unconventional these days, such as using ActiveX controls and updating system files. Some more info on that [here](https://legacyupdate.net/faq/security).

You can always check against [VirusTotal](https://www.virustotal.com/gui/url/c3afda12412693413dc442fb83d06da6de5797d6e637ce259237d447b19d2f83) to see what the consensus is, at the moment 5 of 96 vendors flag it. They’re very generic detections like "phishing", "malicious", "not recommended". These are mostly ones I've had to contact to whitelist Legacy Update in the past. I'll get that done for WUR too. Their engineers should be able to manually confirm it’s safe.

**Comment by TheKnowGuy** [comment]
Ok Thank you so much!

**Comment by CaptainTER06** [comment]
Hi there!
I'm the founder of Windows Update Restored. I can confirm the site is safe to use, it's just using unconventional practices like ActiveX stuff and such as stated by @kirb :-)

**Reply by TheKnowGuy** [reply]
Ok Thank you so much for telling me I might start using it Today, Tomorrow Morning, or evening!

**Reply by TheKnowGuy** [reply]
All I wanted to make sure because last time when I try to use while I’m traveling the ISP blocked me from downloading it
